lib/Star/Kata/Infrastructure/InMemoryInfrastructure.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * This file is part of the phpkata project.
+ *
+ * (c) Yannick Voyer (http://github.com/yvoyer)
+ */
+
+namespace Star\Kata\Infrastructure;
+
+use Star\Kata\KataDomain\KataService;
+use Star\Kata\Model\KataCollection;
+use Star\Kata\Model\KataRepository;
+
+/**
+ * Class InMemoryInfrastructure
+ *
+ * @author  Yannick Voyer (http://github.com/yvoyer)
+ *
+ * @package Star\Kata\Infrastructure
+ */
+final class InMemoryInfrastructure implements KataInfrastructure
+{
+    const CLASS_NAME = __CLASS__;
```

Comment:
Using 5.5, but I just want to support 5.3 and 5.4.
I just don't want to force 5.5, just because of my tests dependencies.